dcos_test_utils/dcos_cli.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import os
+import shutil
+import stat
+import subprocess
+import tempfile
+
+import requests
+
+log = logging.getLogger(__name__)
+
+DCOS_CLI_URL = "https://downloads.dcos.io/binaries/cli/linux/x86-64/latest/dcos"
+
+
+class DcosCli():
+
+    def __init__(self, cli_path):
+        self.path = os.path.abspath(os.path.expanduser(cli_path))
+        updated_env = os.environ.copy()
+        # make sure the designated CLI in on to of the PATH
```

Comment:
I'm having trouble parsing `in on to of the path` - did you miss a word?

Reply:
`is on top of the PATH`?

Reply:
I a word can do not go od

Reply:
"make sure designated CLI is on top of the PATH"